### PR TITLE
Add Github OAuth samples configuration in README.md

### LINKED ISSUE #210: Add sample Github OAuth configuration to README
The current README did not mention how to properly setup Github OAuth in order for new developers to test their new features locally.

## Success criteria
A new developer should be able to setup Github OAuth for local testing with no effort.

### PR DESCRIPTION
**Pull request by akrifari**
Fix #210

### Description
Add Github OAuth samples configuration for new developer who want to contribute

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,19 @@ Short is built on top of [app](https://github.com/byliuyang/app), a reusable fra
...
+### Create Github OAuth Application
+
+1. Register in [Github OAuth](https://github.com/settings/developers) with this following examples:
+
+| Field                      | Value                                                    |
+| -------------------------- | -------------------------------------------------------- |
+| Application Name           | `short`                                                  |
+| Homepage URL               | `https://s.time4hacks.com`                               |
+| Application description    | `URL shortening service written in Go and React`         |
+| Authorization callback URL | `https://s.time4hacks.com/oauth/github/sign-in/callback` |
```
Comment (byliuyang) [inline README.md]: I wonder whether we want to change `https://s.time4hacks.com` to `http://localhost` in order to easy local testing.

File: README.md [hunk 2]
```diff
@@ -23,7 +23,19 @@ Short is built on top of [app](https://github.com/byliuyang/app), a reusable fra
...
+
+2. Once you have registered the application, you should get `Client ID` and `Client Secret`
+3. Now you can copy the `Client ID` and `Client Secret` into their respective places in the `.env` file
```
Comment (byliuyang) [inline README.md]:
```suggestion
3. Now you can copy the `Client ID` and `Client Secret` into their respective locations in the `.env` file
```

File: README.md [hunk 3]
```diff
@@ -23,7 +23,19 @@ Short is built on top of [app](https://github.com/byliuyang/app), a reusable fra
...
 [Create ReCAPTCHA account](http://www.google.com/recaptcha/admin)
 
-[Create Github OAuth App](https://github.com/settings/developers)
+### Create Github OAuth Application
+
+1. Register in [Github OAuth](https://github.com/settings/developers) with this following examples:
+
+| Field                      | Value                                                    |
+| -------------------------- | -------------------------------------------------------- |
+| Application Name           | `short`                                                  |
```
Comment (byliuyang) [inline README.md]:
```suggestion
| Application Name           | `Short`                                                  |
```

File: README.md [hunk 4]
```diff
@@ -23,7 +23,19 @@ Short is built on top of [app](https://github.com/byliuyang/app), a reusable fra
 ### Create reCAPTCHA account
 [Create ReCAPTCHA account](http://www.google.com/recaptcha/admin)
 
-[Create Github OAuth App](https://github.com/settings/developers)
+### Create Github OAuth Application
+
+1. Register in [Github OAuth](https://github.com/settings/developers) with this following examples:
```
Comment (byliuyang) [inline README.md]:
```suggestion
1. Register in [Github OAuth](https://github.com/settings/developers) with the following configurations:
```

**Review by naturetime10 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bigfpanda) [reply README.md]: Yeah, agreed

File: README.md [hunk 5]
```diff
@@ -23,7 +23,19 @@ Short is built on top of [app](https://github.com/byliuyang/app), a reusable fra
 ### Create reCAPTCHA account
 [Create ReCAPTCHA account](http://www.google.com/recaptcha/admin)
 
-[Create Github OAuth App](https://github.com/settings/developers)
+### Create Github OAuth Application
+
+1. Register in [Github OAuth](https://github.com/settings/developers) with the following configurations:
```
Comment (byliuyang) [inline README.md]:
```suggestion
1. Register at [Github OAuth](https://github.com/settings/developers) with the following configurations:
```

**Review by naturetime10 — CHANGES_REQUESTED** [review]

**Comment by byliuyang** [comment]
Hi @bigfpanda , thank you for working on this issue. Please address Code Climate's comments.

**Comment by bigfpanda** [comment]
Thanks :+1:

**Pull request merged**